hyperdevs-team/poeditor-android-gradle-plugin
# Updated dependencies

### PR's key points
- Updated some dependencies (gradle, kotlin, libraries etc)
- Updated Project.xml (automatically done by Android Studio)
- Updated from the deprecated classifier to archiveClassifier

### How to review this PR?
- Check the latest versions of the libs
- Open the project on master with Android Studio and see Project.xml
- https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html#org.gradle.api.tasks.bundling.Jar:classifier

### Definition of Done
- [ ] Tests added (if new code is added) - Does not apply
- [x] There is no outcommented or debug code left
